files/zh-cn/web/javascript/reference/global_objects/intl/numberformat/index.md

```diff
@@ -5,186 +5,112 @@ slug: Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat
-    - `style`
-      - : 要使用的格式样式，默认为“decimal”。
-    - `numberingSystem` {{Experimental_inline}}
-      - : 编号系统。可能的值包括："`arab`"，"`arabext`"，" `bali`"，"`beng`"，"`deva`"，"`fullwide`"，" `gujr`"，"`guru`"，"`hanidec`"，"`khmr`"，" `knda`"，"`laoo`", "`latn`"，"`limb`"，"`mlym`"，" `mong`"，"`mymr`"，"`orya`"，"`tamldec`"，" `telu`"，"`thai`"，"`tibt`"。
-    - `unit`{{Experimental_inline}}
-      - : `unit` 格式中使用的单位，可能的值为在 [UTS #35, Part 2, Section 6](http://unicode.org/reports/tr35/tr35-general.html#Unit_Elements) 定义的核心单元标识符。已从[完整列表](https://github.com/unicode-org/cldr/blob/master/common/validity/unit.xml)中选择了一个[单位子集](https://tc39.es/proposal-unified-intl-numberformat/section6/locales-currencies-tz_proposed_out.html#sec-issanctionedsimpleunitidentifier)以用于 ECMAScript。可以将成对的简单单位与 `“ -per-”` 连接以组成一个复合单位。没有默认值；如果 `style` 为 `“unit”`，必须提供`unit` 属性。
-    - `unitDisplay`{{Experimental_inline}}
-      - : `unit` 格式化中使用的单位格式化样式，默认值为“short”。`“long”` (e.g., `16 litres`)、`“short“` (e.g., `16 l`)、`”narrow“` (e.g., `16l`)
-    - `currency`
-      - : 在货币格式化中使用的货币符号。可能的值是*ISO*的货币代码 (the ISO 4217 currency codes,) 例如 `"USD"` 表示美元，`"EUR"` 表示欧元，或者 `"CNY"是人民币` — 更多请参考 [Current currency & funds code list](http://www.currency-iso.org/en/home/tables/table-a1.html)。没有默认值，如果 `style` 是 `“currency”`，必须提 `currency` 属性。
-    - `currencyDisplay`
-      - : 如何在货币格式化中显示货币。可能的值有 `"symbol"` 表示使用本地化的货币符号，例如 €, `"code"` 表示使用国际标准组织货币代码，`"name"` 表示使用本地化的货币名称，如 `"dollar"`; 默认值是 `"symbol"`.
-    - `useGrouping`
-      - : 是否使用分组分隔符，如千位分隔符或千/万/亿分隔符。可能的值是 true 和 false，默认值是 true。
-
-    下面的属性分为两组：`minimumintegerdigits`，`minimumfractiondigits`，`maximumfractiondigits` 作为一组，`minimumsignificantdigits` 和 `maximumsignificantdigits` 作为一组。如果定义了第二组中的任意一个属性，则忽略第一组的设置。
-
-    - `minimumIntegerDigits`
-      - : 使用的整数数字的最小数目。可能的值是从 1 到 21，默认值是 1.
-    - `minimumFractionDigits`
-      - : 使用的小数位数的最小数目。可能的值是从 0 到 20；默认为普通的数字和百分比格式为 0；默认为货币格式是由 [ISO 4217 currency code list](http://www.currency-iso.org/en/home/tables/table-a1.html) 提供 (如果列表中没有提供则值为 2)。
-    - `maximumFractionDigits`
-      - : 使用的小数位数的最大数目。可能的值是从 0 到 20；纯数字格式的默认值是`minimumfractiondigits` 和 3 中大的那一个；货币格式默认值是`minimumfractiondigits` 和 [ISO 4217 currency code list](http://www.currency-iso.org/en/home/tables/table-a1.html) 中大的那一个 (如果列表中没有提供则值为 2)；百分比格式默认值是 `minimumfractiondigits` 和 0 中大的那一个。
-    - `minimumSignificantDigits`
-      - : 使用的有效数字的最小数目。可能的值是从 1 到 21；默认值是 1。
-    - `maximumSignificantDigits`
-      - : 使用的有效数字的最大数量。可能的值是从 1 到 21；默认是 21.
-    - `notation`{{Experimental_inline}}
-
-      - : 该号码应显示的格式，默认为 `“standard”`。
-
-        - "`standard`" 纯数字格式；
-        - "`scientific`" 返回格式化数字的大小顺序；
-        - "`engineering`" 当被三除时返回十的指数
-        - "`compact`" 代表指数的字符串，默认使用 `“short”` 格式
-
-          - "`compactDisplay`" 仅在 `notation` 为 `“compact”` 时使用，采用 `“short”`（默认）或“`long`”
+<!-- 此交互式示例的源代码存储在 GitHub 存储库中。如果你想为交互式示例项目做出贡献，请克隆 https://github.com/mdn/interactive-examples 并向我们发送拉取请求。 -->
```

Comment:
please remove the hidden notes. We are not adding this anymore. If you like, please also remove this from mdn/content.